codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectAggregator.java

```diff
@@ -48,8 +49,16 @@
  * ...
  * p.addLast("handler", new HttpRequestHandler());
  * </pre>
- * Be aware that you need to have the {@link HttpResponseEncoder} or {@link HttpRequestEncoder}
- * before the {@link HttpObjectAggregator} in the {@link ChannelPipeline}.
+ * Be aware that you need to have the {@link HttpResponseDecoder} or {@link HttpRequestDecoder}
+ * before the {@link HttpObjectAggregator} in the {@link ChannelPipeline} for it to receive
+ * {@link HttpObject}s to aggregate, and you need a {@link HttpResponseEncoder} before the
+ * {@link HttpObjectAggregator} if it will be receiving {@link HttpRequest} messages as the
+ * {@link HttpObjectAggregator} may write a 100 Continue response.
```

Comment:
This makes it sounds like `HttpResponseEncoder` must be before `HttpObjectAggregator` bcz `HttpObjectAggregator` is responsible for sending 100-continue responses.  We defined the relationship in the previous statement and attempting to re-state the relationship in this context is confusing IMO.  Lets just say the two statements independently:

```java
/**
 * Be aware that you need to have the {@link HttpResponseDecoder} or {@link HttpRequestDecoder}
 * before the {@link HttpObjectAggregator} in the {@link ChannelPipeline} for it to receive
 * {@link HttpObject}s to aggregate. {@link HttpObjectAggregator} is also responsible for writing 100
 * Continue responses based upon the max allowed content length.
**/
```